extensions/amp-moviads-player/0.1/test/validator-amp-moviads-player.out

```diff
@@ -0,0 +1,21 @@
+PASS
```

Comment:
Thanks for adding a copyright block to the test input file. Can you run the validator with update-tests to pick up the changes here.

Reply:
I made changes to the test